scheduler/pmc-processor-task/src/main/kotlin/ac/uk/ebi/pmc/data/MongoRepository.kt

```diff
@@ -37,4 +46,11 @@ class MongoRepository(
         collection.insertOne(FileDoc(file.name, file.absolutePath, accNo))
         return collection.findOne(Filters.eq("path", file.absolutePath))!!.id
     }
+
+    suspend fun getSubFiles(ids: List<ObjectId>): MutableList<Option<FileDoc>> {
```

Comment:
I think we should not retrieve a list of options , I think there must be a method that can be used when we are sure element exists

Reply:
Fixed